--- InsightToolkit-4.13.3.orig/Modules/Core/Common/include/itkGaussianOperator.hxx	2020-07-24 01:49:56.000000000 +0000
+++ InsightToolkit-4.13.3/Modules/Core/Common/include/itkGaussianOperator.hxx	2021-01-19 12:12:30.390000000 +0000
@@ -55,7 +55,7 @@
       }
     if ( coeff.size() > m_MaximumKernelWidth )
       {
-      itkWarningMacro("Kernel size has exceeded the specified maximum width of "
+      itkDebugMacro("Kernel size has exceeded the specified maximum width of "
                       << m_MaximumKernelWidth << " and has been truncated to "
                       << coeff.size()
                       << " elements.  You can raise the maximum width using the SetMaximumKernelWidth method.");
