--- InsightToolkit-4.13.3.orig/Modules/Core/Common/include/itkUnaryFunctorImageFilter.hxx	2020-07-24 01:49:56.000000000 +0000
+++ InsightToolkit-4.13.3/Modules/Core/Common/include/itkUnaryFunctorImageFilter.hxx	2021-01-19 12:12:30.410000000 +0000
@@ -61,6 +61,8 @@
     return;
     }
 
+
+  outputPtr->CopyInformation(inputPtr);
   // Set the output image largest possible region.  Use a RegionCopier
   // so that the input and output images can be different dimensions.
   OutputImageRegionType outputLargestPossibleRegion;
