diff -burN InsightToolkit-4.7.2.orig/Modules/Core/Common/include/itkUnaryFunctorImageFilter.hxx InsightToolkit-4.7.2/Modules/Core/Common/include/itkUnaryFunctorImageFilter.hxx
--- InsightToolkit-4.7.2.orig/Modules/Core/Common/include/itkUnaryFunctorImageFilter.hxx	2017-12-20 14:31:15.000000000 +0100
+++ InsightToolkit-4.7.2/Modules/Core/Common/include/itkUnaryFunctorImageFilter.hxx	2017-12-20 14:33:47.000000000 +0100
@@ -61,6 +61,8 @@
     return;
     }
 
+  outputPtr->CopyInformation(inputPtr);
+  
   // Set the output image largest possible region.  Use a RegionCopier
   // so that the input and output images can be different dimensions.
   OutputImageRegionType outputLargestPossibleRegion;
