diff -burN OTB-6.0.1.orig/Modules/Core/ImageBase/include/otbMultiChannelExtractROI.txx OTB-6.0.1/Modules/Core/ImageBase/include/otbMultiChannelExtractROI.txx
--- OTB-6.0.1.orig/Modules/Core/ImageBase/include/otbMultiChannelExtractROI.txx	2017-12-12 15:02:41.000000000 +0100
+++ OTB-6.0.1/Modules/Core/ImageBase/include/otbMultiChannelExtractROI.txx	2017-12-12 15:21:17.000000000 +0100
@@ -303,14 +303,14 @@
     unsigned int channelOut(0);
     unsigned int nbChannels(0);
 
-    InputImagePixelType pixelInput;
-    while (!outIt.IsAtEnd())
-      {
       OutputImagePixelType pixelOutput;
       pixelOutput.Reserve(outputPtr->GetVectorLength());
-      pixelInput = inIt.Get();
+    const unsigned int l_NbChannelsWorks = m_ChannelsWorks.size();
+    while (!outIt.IsAtEnd())
+      {
+      InputImagePixelType const& pixelInput = inIt.Get();
       channelOut = 0;
-      for (nbChannels = 0; nbChannels < m_ChannelsWorks.size(); ++nbChannels)
+      for (nbChannels = 0; nbChannels < l_NbChannelsWorks; ++nbChannels)
         {
         channelIn = m_ChannelsWorks[nbChannels] - 1;
         pixelOutput[channelOut] = static_cast<OutputValueType>(pixelInput[channelIn]);
@@ -319,7 +319,6 @@
       outIt.Set(pixelOutput);
       ++outIt;
       ++inIt;
-      progress.CompletedPixel();
       }
     }
 }
