--- hdf-4.2.15.orig/configure
+++ hdf-4.2.15/configure
@@ -23670,7 +23670,6 @@
       if test -z "$HAVE_OLDRPC" ; then
         LIBS="$LIBS -ltirpc"
         CPPFLAGS="$SYSCPPFLAGS -I/usr/include/tirpc"
-        unset ac_cv_header_rpc_rpc_h
         ac_fn_c_check_header_mongrel "$LINENO" "rpc/rpc.h" "ac_cv_header_rpc_rpc_h" "$ac_includes_default"
 if test "x$ac_cv_header_rpc_rpc_h" = xyes; then :
   :
