diff -burN gdal-2.4.1_orig/swig/python/GNUmakefile gdal-2.4.1/swig/python/GNUmakefile
--- gdal-2.4.1_orig/swig/python/GNUmakefile	2019-03-15 13:30:02.000000000 +0100
+++ gdal-2.4.1/swig/python/GNUmakefile	2020-04-27 18:32:07.572163441 +0200
@@ -3,9 +3,17 @@
 include ../../GDALmake.opt

 ifndef PYTHON
+	echo "dans if"
         PYTHON=python
 endif

+echo "=======toto"
+echo "$(PYTHON)"
+
+ifndef DESTDIR
+    DESTDIR=/
+endif
+
 all: build

 BINDING = python